addon/doc/es/README.md

```diff
@@ -17,12 +17,12 @@ La última versión está disponible para [descargar en este enlace](https://dav
 
 # requisitos.
 ## NVDA.
-  Necesitas NVDA 2018.4 o posterior. Este driver es compatible con python 3, así que podrás usarlo con versiones futuras de NVDA. Una vez que NVDA con python 3 sea liberada, este driver dejará de ser compatible con python 2.7. Por favor usa las últimas versiones de NVDA. ¡Es gratis!
+  Necesitas NVDA 2019.3 o posterior.
 
 ## Las librerías del sintetizador IBMTTS.
   Esto es solo el controlador, debes buscar las librerías en otro lugar.  
   El controlador soporta las librerías ligeramente más recientes que añaden el soporte del idioma este-asiático, y tiene correcciones específicas para la codificación adecuada del texto. Sin embargo, las librerías más antiguas sin esto deberían funcionar.  
-  A partir de la versión 21.01, el controlador también soporta la integración adecuada con los binarios aún más nuevos de IBM, en lugar de solo los binarios de  SpeechWorks. Se incluye un conjunto de correcciones independientes para ese controlador, y se tienen en cuenta los idiomas adicionales y otras diferencias. Solo se apoyan las voces formantes en la actualidad. Gracias a @mohamed00 por este trabajo.
+  A partir de la versión 21.01, el controlador también soporta la integración adecuada con los binarios aún más nuevos de IBM, en lugar de solo los binarios de  SpeechWorks. Se incluye un conjunto de correcciones independientes para ese controlador, y se tienen en cuenta los idiomas adicionales y otras diferencias. Solo se apoyan las voces formantes en la actualidad. Gracias a @mohamed00 por este trabajo. Tenga en cuenta que cuando se utiliza las librerías de IBM, debes deshabilitar la opción Enviar siempre la configuración de voz actual.
```

Comment:
Can you please update the version numbers mentioned in the translations? 21.01 is a referenced to an old fork which I have removed recently, support for IBM binaries was introduced in this repository in version 23.01A1.